kafka-quarkus-call-center/src/main/java/org/acme/callcenter/rest/CallCenterResource.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.acme.callcenter.rest;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.acme.callcenter.data.CallCenterDataGenerator;
+import org.acme.callcenter.domain.CallCenter;
+import org.acme.callcenter.service.SimulationService;
+import org.acme.callcenter.service.SolverService;
+
+@Path("/call-center")
```

Comment:
In TimeTableResource we use camelCase for this too

```suggestion
@Path("/callCenter")
```

If we go non-camel case for URL's, then the other quickstarts should do the same.

Reply:
I quite agree with the argument [here](https://restfulapi.net/resource-naming/) that using hyphens instead of the camelCase improves readability. Look at the URL of this page. It's `optaplanner-quickstarts` not `optaplannerQuickstarts`.

+1 to change to this convention in the other quickstarts, if we seek absolute consistency.